api-mock/index.js

```diff
@@ -0,0 +1,133 @@
+let EmailPassword = require("supertokens-node/recipe/emailpassword");
```

Comment:
can we make this TS?

Reply:
yes but we will need to run a build first or figure it out on how mocha runs ts